code/modules/food_and_drink/cakes.dm

```diff
@@ -641,6 +641,19 @@
 
 #endif
 
+/obj/item/reagent_containers/food/snacks/cake/bacon
```

Comment:
intentional? doubtful.

Reply:
Yup, totally unintentional. Was an artifact of trying to merge the new cake patch cake patch modular cake patch without fully understanding what it changed.